src/octoprint/plugins/classicwebcam/__init__.py

```diff
@@ -0,0 +1,261 @@
+__license__ = "GNU Affero General Public License http://www.gnu.org/licenses/agpl.html"
+__copyright__ = "Copyright (C) 2020 The OctoPrint Project - Released under terms of the AGPLv3 License"
+
+import threading
+
+import requests
+from flask_babel import gettext
+
+import octoprint.plugin
+from octoprint.schema.config.webcam import RatioEnum, Webcam, WebcamCompatibility
+
+
+class ClassicWebcamPlugin(
+    octoprint.plugin.AssetPlugin,
+    octoprint.plugin.TemplatePlugin,
+    octoprint.plugin.SettingsPlugin,
+    octoprint.plugin.WebcamPlugin,
+    octoprint.plugin.WizardPlugin,
+):
+    def __init__(self):
+        self._capture_mutex = threading.Lock()
+
+    # ~~ TemplatePlugin API
+
+    def get_assets(self):
+        return {
+            "js": [
+                "js/classicwebcam.js",
+                "js/classicwebcam_settings.js",
+                "js/classicwebcam_wizard.js",
+            ],
+            "less": ["less/classicwebcam.less"],
+            "css": ["css/classicwebcam.css"],
+        }
+
+    def get_template_configs(self):
+        return [
+            {
+                "type": "settings",
+                "template": "classicwebcam_settings.jinja2",
+                "custom_bindings": True,
+            },
+            {
+                "type": "webcam",
+                "name": "Classic Webcam",
+                "template": "classicwebcam_webcam.jinja2",
+                "custom_bindings": True,
+                "suffix": "_real",
+            },
+            {
+                "type": "webcam",
+                "name": "Classic Webcam 2",
+                "template": "classicwebcam_webcam_2.jinja2",
+                "custom_bindings": False,
+                "suffix": "_mock",
+            },
+            {
+                "type": "wizard",
+                "name": "Classic Webcam Wizard",
+                "template": "classicwebcam_wizard.jinja2",
+                "suffix": "_wizard",
+            },
+        ]
+
+    # ~~ WebcamPlugin API
+
+    def get_webcam_configurations(self):
+        streamRatio = self._settings.get(["streamRatio"])
+        if streamRatio == "4:3":
+            streamRatio = RatioEnum.four_three
+        else:
+            streamRatio = RatioEnum.sixteen_nine
+        webRtcServers = self._settings.get(["streamWebrtcIceServers"])
+        cacheBuster = self._settings.get_boolean(["cacheBuster"]) is True
+        stream = self._get_stream_url()
+        snapshot = self._get_snapshot_url()
+        flipH = self._settings.get_boolean(["flipH"]) is True
+        flipV = self._settings.get_boolean(["flipV"]) is True
+        rotate90 = self._settings.get_boolean(["rotate90"]) is True
+        snapshotSslValidation = (
+            self._settings.get_boolean(["snapshotSslValidation"]) is True
+        )
```

Comment:
`get_boolean` is already guaranteed to return a boolean, and checking whether that `is True` to set `True` on a var is a no-op.

```suggestion
        cacheBuster = self._settings.get_boolean(["cacheBuster"])
        stream = self._get_stream_url()
        snapshot = self._get_snapshot_url()
        flipH = self._settings.get_boolean(["flipH"])
        flipV = self._settings.get_boolean(["flipV"])
        rotate90 = self._settings.get_boolean(["rotate90"])
        snapshotSslValidation = (
            self._settings.get_boolean(["snapshotSslValidation"])
        )
```

Reply:
Better save than sorry :D Removed